tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 - name: install
   apt:
     name: "{{ item }}"
-    state: latest
+    state: present
```

Comment:
This change is not backwards compatible, can you make it configurable. Let's add a variable called `fail2ban_install_state`, with `latest` as the default. What do you think?